cluster/saltbase/salt/kube-addons/init.sls

```diff
@@ -165,6 +165,28 @@ addon-dir-create:
     - file_mode: 644
 {% endif %}
 
+{% if pillar.get('enable_pod_security_policy', '').lower() == 'true' %}
+/etc/kubernetes/addons/podsecuritypolicy:
```

Comment:
`.../addons/podsecuritypolicies`?

Reply:
Actually, I have no idea where this is actually used, but changed anyway.